src/main/java/io/github/classgraph/ClassGraph.java

```diff
@@ -1429,6 +1431,31 @@ public ClassGraph enableMemoryMapping() {
         return this;
     }
 
+    /**
+     * If true, provide all versions of a multi-release resource using their multi-release path prefix, instead of
+     * just the one the running JVM would select. Implicitly disables {@link #enableClassInfo()} and all features
+     * depending on it.
+     * 
+     * @return this (for method chaining).
+     */
+    public ClassGraph enableMultiReleaseVersions() {
+        scanSpec.enableMultiReleaseVersions = true;
+
+        scanSpec.enableClassInfo = false;
```

Comment:
I wasn't sure whether this was the idiomatic approach or whether `enableMultiReleaseVersions()` should just throw an `IllegalArgumentException` if `enableClassInfo()` was set and vice-versa.

Reply:
I think this behavior is fine, given that you documented it.